src/InstagramClone.Api/Startup.cs

```diff
@@ -13,26 +15,47 @@ public static WebApplication InitialIzeApp(string[] args)
             Configure(app);
             return app;
         }
-       
+
 
         private static void ConfigureService(WebApplicationBuilder builder)
         {
             
+            builder.Services.AddControllers(config =>
+            {
+                config.RespectBrowserAcceptHeader = true;
+                config.ReturnHttpNotAcceptable = true;
+            }).AddXmlDataContractSerializerFormatters();
+
+             builder.Services.AddControllers();
+            builder.Services.AddCors();
+
```

Comment:
You didn't add comments here to explain this line

Reply:
Ok. Done